app/controllers/catalog_controller.rb

```diff
@@ -104,6 +104,10 @@ class CatalogController < ApplicationController
     # and we use them when users click on the "Keywords" links in the Show page.
     config.add_facet_field 'subject_all_ssim', label: 'Keywords', show: false
 
+    # An extra facet to filter DataSpace vs PDC Describe records at will
+    # (this is handy during the migration)
+    config.add_facet_field 'data_source_ssi', label: 'Source'
+
```

Comment:
This field is not strictly necessary for the user, but it helps us (devs) a lot when troubleshooting the data since it allows us to filter DataSpace vs PDC-Describe records at will and validate the data. We can remove it once the migration has been completed since by then there will be a single source value (PDC-Describe_